Jenkinsfile

```diff
@@ -120,8 +121,35 @@ pipeline {
         container('base') {
           nxWithGitHubStatus(context: 'release', message: 'Release') {
             script {
-              // tag the version - nuxeo-helm-chart doesn't follow the v1.2.3 convention
-              nxGit.commitTagPush(tag: env.VERSION)
```

Comment:
Does this mean we are switching to v3.x.y?

Reply:
Yes it is.
I also renamed all previous tags to this pattern.
I didn't see any impact on such, we're just losing the tag date, but not a big deal as we have the tagged commit date.